src/server/routes/api/document/pubmed/searchPubmed.js

```diff
@@ -51,6 +54,86 @@ const eSearchPubmed = ( term, opts ) => {
   .then( pubmedDataConverter );
 };
 
+/**
+ * Search for an article using Crossref and return a Pubmed-style result.
+ * 
+ * This approach uses Crossref to find the article's DOI.  The DOI is sent to PMC to find a
+ * PMID.  If the PMC query fails for any reason, a query to Pubmed Eutils is made using the 
+ * DOI as the search term.  For some articles, PMC fails.  For others, Eutils fails.  Using
+ * both gives good coverage.
+ * 
+ * @param {String} term The search term to find the article (e.g. title)
+ * @param {Object} opts Pubmed API options
+ * @returns {Object} A Pubmed search result object
+ */
+const crSearchPubmed = async ( term, opts ) => {
+  const doi = await crSearchForDoi(term);
+
+  const pmid = await getPmidFromDoi(doi);
+
+  if( pmid ){
+    return {
+      searchHits: [ pmid ],
+      count: 1,
+      query_key: '-1',
+      webenv: 'imitated_pubmed_format_from_crossref_query'
+    };
+  } else {
+    return eSearchPubmed(doi, opts);
+  }
+};
+
+/**
+ * Search for an article's DOI using Crossref
+ * @async
+ * @param {String} term The search term (e.g. article title)
+ * @returns {String} The DOI, null if not found
+ */
+const crSearchForDoi = async ( term ) => {
+  const params = {
+    'query.bibliographic': term
```

Comment:
- filter on type: `filter=type:journal-article`
- also `rows=1`

Reply:
NB: `filter=type:journal-article` will exclude `"subtype": "preprint"` i.e. bioRXiv.

Reply:
Pubmed also has preprint articles, though perhaps not as quickly as Biorxiv.  Would this filter affect those articles?

Reply:
Oh boy that's news to me. I need to do some homework : https://www.ncbi.nlm.nih.gov/pmc/about/nihpreprints/

Reply:
I think COVID-19 motivated them to push that project forward.  There's a little marker on Pubmed search results for preprint articles.